docs/copied-from-beats/command-reference.asciidoc

```diff
@@ -16,23 +16,22 @@
 
 :export-command-short-desc: Exports the configuration or index template to stdout
 :help-command-short-desc: Shows help for any command
+:keystore-command-short-desc: Manages the <<keystore,secrets keystore>>
 :modules-command-short-desc: Manages configured modules
 :run-command-short-desc: Runs {beatname_uc}. This command is used by default if you start {beatname_uc} without specifying a command
-
-ifeval::["{has_ml_jobs}"=="yes"]
```

Comment:
Please don't remove this, there is a PR https://github.com/elastic/beats/pull/6184/files open in beats that introduces this distinction, as it doesn't make sense to describe ML here for APM.

Reply:
I copied over a fresh version of the command-reference that has an entry for the keystore.
Once that PR gets merged and copied over, we will have the right thing, no?

I think is better to wait for that PR to get merged first, than manually editing copied files.

Reply:
As soon as this gets merged, it is in the official docs. 

Thus, I'd not suggest to merge in some documentation that is not supposed to show up for APM Server, but rather to copy over the part for the keystore as is and leave the other part of the file untouched for now.